vmpooler_client/command_parser.py

```diff
@@ -110,6 +146,8 @@ def add_sub_command(self, parent, sub_cmd_name, desc, func):
 
     if parent not in self._commands:
       raise KeyError("The '{}' parent command has not been defined yet!".format(parent))
+    elif isinstance(self._commands[parent], argparse.ArgumentParser):
```

Comment:
Ooh, tricky. Do you know what class it'll be if it's not a `argparse.ArgumentParser`? Might be worth putting a comment in explaining it for future generations